src/api/extensions.ts

```diff
@@ -13,4 +15,4 @@ router.get('/', async (req, res) => {
   }
 })
 
-module.exports = router
+export = router
```

Comment:
I've never seen this convention before. Should it be `export default router`? or is this a TS thing?

Reply:
ah - I think it's not a default export. Usually I'd do something like:

`export const router = Router()`

Reply:
It's a TS thing yeah. Apparently it's the equivalent of CommonJS's `module.exports` object:
https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require